### ISSUE TITLE
proxy server

### ISSUE THREAD

**Issue by pmatuszy** [mxpv/podsync#213]
Hi

It used to be the option in the config file to use proxy server (i've been running this for over a year):
```
[server]
port = 8000
hostname = "http://DOESNTMATTER:8080"
**proxy = "socks4://127.0.0.1:9050"**
data_dir = "/podsync-hdd"

```
Is it missing now? I just installed new podsync selft-hosted version, compiled it and when I try to start it I have this in my logfile:

`time="2021-01-28T22:42:04Z" level=fatal msg="failed to load configuration file" error="failed to unmarshal toml: line 4: field corresponding to `proxy' is not defined in config.Server"
`
Could you confirm that this option is now missing?

Paul

**Comment by matejdro** [comment]
See https://github.com/mxpv/podsync/issues/201#issuecomment-748151495

**Comment by mxpv** [comment]
Fixed in v2.4.1
